### ISSUE TITLE
Extract pretrained LSTM output?

### ISSUE THREAD

**Issue by ruizhaogit** [tflearn/tflearn#219]
Hi TFlearn Team,

How can I extract the output of a trained LSTM model?

My code is like this:

net = tflearn.input_data(shape=[None, 300, 150])
lstm_output = tflearn.lstm(net, 128, return_seq=False) 
net = tflearn.fully_connected(lstm_output, 10, activation='softmax')
net = tflearn.regression(net, loss='softmax_categorical_crossentropy', name="output1") 
model = tflearn.DNN(net)
model.load("/model_01/-2000")

I want to get the "lstm_output", does anyone know, how I can do this?

Thank you!

Best,
Rui
